index.d.ts

```diff
@@ -67,3 +67,11 @@ export declare class SessionWrapper {
   static createSession(options: SessionOptions): Promise<SessionWrapper>
   queryUnpagedNoValues(query: string): Promise<QueryResultWrapper>
 }
+export declare class LocalTime {
+  value: bigint
+  hour: number
+  minute: number
+  second: number
+  nanosecond: number
+  static new(totalNanoseconds: bigint): LocalTime
+}
```

Comment:
About `Adding draft of local-time implementation in Rust and new-utils.js from adespawn.` commit:
If the contents of the commit were done by someone else, but it was you who commited it, you can add `Co-authored-by: <name> <email>` annotation to the commit message (as the last line).